authenticator7

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+dataset=pd.read_csv('users.csv')
```

Comment:
The goal here is to define an `Authenticator` class. Try looking at [the docs on custom authenticators](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html#how-to-write-a-custom-authenticator) and [DummyAuthenticator](https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/auth.py#L691) as an example.

This class should:

1. have a `.authenticate()` method
2. check username and password against its csv file
3. return the username on successful authentication,
4. return None on unsuccessful authentication